Keboola/DataLoader/ConfigValidator.php

```diff
@@ -143,10 +143,30 @@ private function validateConfig()
         $this->type = $rowData['configuration']['type'];
         $this->input['tables'] = $rowData['configuration']['input'];
         foreach ($this->input['tables'] as &$table) {
-            $table['where_values'] = $table['where_values'] ?? $table['whereValues'] ?? null;
-            $table['where_column'] = $table['where_column'] ?? $table['whereColumn'] ?? null;
-            $table['where_operator'] = $table['where_operator'] ?? $table['whereOperator'] ?? null;
-            $table['changed_since'] = $table['changed_since'] ?? $table['changedSince'] ?? null;
+            if (count($table['whereValues']) > 0) {
+                $table['where_values'] = $table['whereValues'];
+            }
+            if (isset($table['whereValues'])) {
```

Comment:
ten isset je tu nanic ne?

Reply:
jo, jenom jsem ty fieldy v `$table` chtěl pročistit, ať tam vedle sebe nejsou `whereValues` a `where_values`

Reply:
to chapu, ale staci dat unset

Reply:
aha! to jsem nevěděl, jasně, upravím.